input/pagecontent/index.md

```diff
@@ -39,13 +39,15 @@ Let's start with a simple example, defining a "patient_demographics" view with t
 }
 ```
 
-This will result in a table like this, which can be persistend and queried in your database of choice:
+This will result in a "patient_demographics" table that looks like this. The table can be persistend and queried
```

Comment:
Small change: "persistend" to "persisted"

Reply:
Done.